ping/main.go

```diff
@@ -13,7 +13,7 @@ import (
 
```

Comment:
I wonder whether we still need the normal ping example now that we have the ping-interop example. No strong opinion, though might be worth deduplicating.

Reply:
Agreed, fixed!